### PR TITLE
Fix glossary section

### PR DESCRIPTION
**Pull request by WangCheng0116**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -412,14 +418,16 @@ Then, use the `load` command to import the data.
...
+| **GUI**                               | A graphical user interface (GUI) is a digital interface in which a user interacts with graphical components such as icons, buttons, and menus.                                                                                                                                                                                   |
+| **JSON (JavaScript Object Notation)** | JSON is a lightweight data format commonly used for representing structured data.                                                                                                                                                                                                                                                |
+| **Level**                             | The level represents the proficiency with a specific flashcard.                                                                                                                                                                                                                                                                  |
```
Comment (itsNatTan) [inline docs/UserGuide.md:431]: Perhaps add a line saying lower means less proficient or smth to that effect

**Review by itsNatTan — CHANGES_REQUESTED** [review]
Please add that clarification. Otherwise lgtm

File: docs/UserGuide.md (hunk 1 above)
Comment (WangCheng0116) [reply docs/UserGuide.md]: i have mentioned this in this [commit](https://github.com/AY2324S1-CS2103T-T11-4/tp/pull/316/commits/68f13cc3a2f5b4583f7d936c37aeed09c12e3270) but i guess you are right. It has been fixed.

**Comment by itsNatTan** [comment]
Lgtm

**Review by itsNatTan — APPROVED** [review]
LGTM thanks for changing

**Pull request merged**